packages/sit-onyx/src/components/OnyxSwitch/OnyxSwitch.vue

```diff
@@ -184,10 +191,16 @@ $input-width: calc(2 * $icon-size - 2 * $container-padding);
         background-color var(--onyx-duration-sm) ease;
       overflow: hidden;
       color: var(--onyx-color-text-icons-neutral-soft);
+      height: $icon-size;
+      width: $icon-size;
 
       .onyx-icon {
         --icon-size: #{$icon-size};
       }
+
+      .onyx-switch__spinner {
+        color: var(--onyx-color-text-icons-neutral-soft);
```

Comment:
```suggestion
        padding: var(--onyx-1px-in-rem);
```

The loading indicator inherits the font color so since we already set it in "onyx-switch__icon" we don't need to define it here. But imho we should add a small padding so the indicator is better visible :)

Reply:
In figma it also has a padding/gap :)